src/utils/api.js

```diff
@@ -31,6 +33,98 @@ export const formatActions = (actionsByDayObject) => {
   });
 };
 
+// Takes array of action objects and returns an object with {key: value} pairs of {hourOfDay: #-of-actions}
+export const getActionsByTimeOfDay = (actions) => {
+  const actionsByTimeOfDay = {
+    'Late night': 0,
+    'Early morning': 0,
+    Morning: 0,
+    Afternoon: 0,
+    Evening: 0,
+    Night: 0,
+  };
+  actions.forEach((action) => {
+    const actionHourOfDay = action.createdAt.slice(11, 13);
+    if (actionHourOfDay >= 0 && actionHourOfDay <= 4) {
```

Comment:
I would probably use `>` for consistency with the rest of the clauses.

Reply:
but then we wouldn't have logic for the case `actionHourOfDay === 0`...

Reply:
Yes, you're right. Maybe go the opposite way? `>= && <`, and use 5, 9, 13, 16, 21, 24?